service/src/test/java/greencity/filters/RatingStatisticsSpecificationTest.java

```diff
@@ -180,4 +193,22 @@ void toPredicate() {
         verify(criteriaBuilderMock).and(andDataRangePredicate, andCurrentRatingPredicate);
 
     }
+
+    @Test
+    void toPredicate_exceptionTest() {
+        initRatingStatisticsViewDto("2", "NOTREAL", "1", "", "2021-01-12", "2021-01-13", "", "50");
+        init();
```

Comment:
why do we need these methods in this test?
we call them in `setUp`

Reply:
fixed